input/resources/structuredefinition-result-copies-to.xml

```diff
@@ -2,7 +2,7 @@
 <StructureDefinition xmlns="http://hl7.org/fhir">
   <id value="result-copies-to" />
   <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
-    <valueInteger value="0"/> 
+    <valueInteger value="1"/> 
```

Comment:
Missing change log entry for this change.

Reply:
Added, thanks.

Reply:
Confirmed.